personal_data/avatar/README.md

```diff
@@ -11,3 +12,18 @@ https://commons.wikimedia.org/wiki/File:Benjamin_Franklin_1706-1790.jpg
 The image was animated by this service:
 
 https://www.myheritage.de/deep-nostalgia/
+
+# New features
+
+Added the module avatar-generator-GUI inside AI-replica/tools folder.
+Added the URL to Google Colab Jupiter Notebook
+with the procedure for generate the animated 
+pictures for realistic face expression.
+
+# Just follow the instructions.
+
+Simply copy the Colab in your Drive.
+Then enter in your Drive and open your Colab.
+Then apply the inside procedure step by step.
+
+[Colab Jupyter's notebook](https://colab.research.google.com/drive/1x-CnBKoaLCBpFQ2LMURYyoCuytY6yifb#scrollTo=IcMX7ueZO0Oa)
```

Comment:
It seems that one can access the colab only after requesting an access. To make it future-proof, I would suggest to make the colab openly accessible if possible.